quadtree/quadmap_test.go

```diff
@@ -32,31 +32,30 @@ func TestCreateTileAtSlippyCoords(t *testing.T) {
 	err := qm.AddTile(tile)
 	assert.Nil(t, err, "Should not have error when adding tile")
 
-	// quadindex for 1,1,1 is 13835058055282163713
+	// quadindex for 1,1,1 is 0xc000000000000001
```

Comment:
can change to binary?   0b1100000000000000000000000000000000000000000000000000000000000001